src/pyuvdata/utils/io/hdf5.py

```diff
@@ -679,6 +679,20 @@ def antenna_names(self) -> list[str]:
             self.header["antenna_names"][:].astype("|S"), encoding="utf8"
         )
 
+    @cached_property
+    def mount_type(self) -> list[str]:
+        """The antenna names in the file."""
+        return np.char.decode(
+            self.header["mount_type"][:].astype("|S"), encoding="utf8"
+        )
+
+    @cached_property
+    def feed_array(self) -> np.ndarray[str]:
+        """The antenna names in the file."""
+        return np.char.decode(
+            self.header["feed_array"][:].astype("|S"), encoding="utf8"
+        ).astype(np.object_)
```

Comment:
should feed_angle be here too?

Reply:
No -- this only needs to be here b/c of string handling.